torch/ao/quantization/pt2e/qat_utils.py

```diff
@@ -715,19 +717,20 @@ def _fold_conv_bn_qat_helper(
         [True, False],  # is_per_channel
         [True, False],  # has_bias
         [True, False],  # bias_is_quantized
+        [True, False],  # bn_is_training
```

Comment:
We don't need to do this because all matching uses `ignore_literals`, so this doesn't actually do anything